recipe/meta.yaml

```diff
@@ -38,4 +38,5 @@ about:
 
 extra:
   recipe-maintainers:
```

Comment:
Also... shouldn the `source.sha256` be hardcoded as it is currently? 🤔 what does it correspond to?

Reply:
The sha256 is set via the autotick bot, and corresponds to the sha of the package you've uploaded to pypi. 
It's detected automatically, please see eg:

https://github.com/conda-forge/git-machete-feedstock/pull/25

Reply:
You don't need to drive version updates or package updates yourself.
The autotick bot will detect when you cut a new pypi release, open a PR here for the version bump, and merge it.